src/js/containers/search/SearchContainer.jsx

```diff
@@ -184,6 +197,12 @@ const SearchContainer = ({ history }) => {
         }
     }, [appliedFilters]);
 
+    useEffect(() => {
+        if (request.current) {
```

Comment:
Same, this is going to fire onMount. We want to use cleanup. See documentation link above!